CHANGELOG.md

```diff
@@ -1,3 +1,190 @@
+## Release 0.14.0
+
+Highlights:
+
+This release comes with two main new features:
+  * Plugins. Now you can create your http-based plugin to provide new metrics
+    and display them in Scope. You can read more about it and see some examples [here]().
+  * _Metrics on canvas_: Metrics are now displayed on the nodes and not just on
```

Comment:
I agree about the colon. On the other hand, the italics are due to _Metrics on canvas_ being a term we have coined ourselves.